src/sinks/splunk_hec/metrics/config.rs

```diff
@@ -57,18 +72,29 @@ pub struct HecMetricsSinkConfig {
     /// The name of the index where to send the events to.
     ///
     /// If not specified, the default index is used.
+    #[configurable(metadata(docs::examples = "{{ host }}", docs::examples = "custom_index"))]
     pub index: Option<Template>,
 
     /// The sourcetype of events sent to this sink.
     ///
     /// If unset, Splunk will default to `httpevent`.
+    #[configurable(metadata(
+        docs::examples = "{{ sourcetype }}",
+        docs::examples = "_json",
+        docs::examples = "httpevent"
+    ))]
     pub sourcetype: Option<Template>,
 
     /// The source of events sent to this sink.
     ///
     /// This is typically the filename the logs originated from.
     ///
     /// If unset, the Splunk collector will set it.
```

Comment:
worthy of an external link? 🤷

Reply:
Same as [other PR](https://github.com/vectordotdev/vector/pull/16334#discussion_r1101266431).